1-exercises/A-reading-list/script.js

```diff
@@ -1,5 +1,27 @@
 function readingList(books) {
-  // Write your code here...
+ 
+const divEl = document.getElementById("content")
+const ulEl = document.createElement("ul")
+
+ for(let book of books){
```

Comment:
```suggestion
 for(const book of books){
```
Identifier `book` isn't re-assigned in this for loop